1-development/user-changes-challenge/scripts/createSchema.sql

```diff
@@ -0,0 +1,10 @@
+\connect ixchallenge postgres;
+
+drop table if exists changes;
+
+create table changes (
+    uuid serial primary key,
+    userID integer not null,
+    date timestamp not null default now(),
+    diff json not null
```

Comment:
```Advantages``` - Easier to query by the database, instead of loading the object to memory and then query it (which would make it less efficient).
```Disadvantages``` - More requests to the database (1 per different name of the changed field), and harder algorithm to implement in SQL than in Javascript.

I only used PostgreSQL to interact with this database.